baram/iam_manager.py

```diff
@@ -61,3 +62,32 @@ def list_group_policies(self, user_group_name, max_items=100):
         :return:
         '''
         return self.cli.list_group_policies(GroupName=user_group_name, MaxItems=max_items)
+
+    def get_policies(self, scope: str = 'All', max_result: int = 1000):
+        """
+        Lists all policies in IAM
+        :param scope: 'Local' for customer managed policies, 'AWS' for AWS managed policies, 'All' for all policies
+        :param max_result: max number of results (max=1000)
+        :return:
+        """
+        policies = self.cli.list_policies(Scope=scope, MaxItems=max_result)
+        result = policies['Policies']
+        while policies['IsTruncated']:
+            marker = policies['Marker']
+            next_policies = self.cli.list_policies(Scope=scope, MaxItems=max_result, Marker=marker)
+            result += next_policies['Policies']
+            if 'Marker' not in next_policies:
+                break
+        return result
```

Comment:
I think `policies` isn't renewed in this loop.

Reply:
Also, maybe this method return result with many repeated values. So some suitable assertions maybe helpful for this.